activation/activation.go

```diff
@@ -67,32 +68,27 @@ type Config struct {
 // it is responsible for initializing post, receiving poet proof and orchestrating nipst. after which it will
 // calculate total weight and providing relevant view as proof.
 type Builder struct {
-	started atomic.Bool
-
 	eg errgroup.Group
 
 	signer           *signing.EdSigner
 	accountLock      sync.RWMutex
-	nodeID           types.NodeID
 	coinbaseAccount  types.Address
 	goldenATXID      types.ATXID
 	regossipInterval time.Duration
 	cdb              *datastore.CachedDB
+	localDB          *datastore.LocalDB
```

Comment:
please use sql instance directly. datastore module will be removed

Reply:
I moved the code to the `localsql` package and made it easier to use.